location/templates/location.html

```diff
@@ -46,14 +46,13 @@ <h2>Apartments</h2>
             <ul>
             {% for apt in zillow_list %}
```

Comment:
Once we get rid of zillow information in the location page, I think we can iterate over object.apartment_set instead of zillow_list

Reply:
Let's do it later